### PR TITLE
Support image long press handler

### PR DESCRIPTION
**Pull request by nojimage**
Add the `onLongPress` and `delayLongPress` options.

I want to bring up the operation menu by long press the image, so I added.

### CODE REVIEW

File: src/components/ImageItem/ImageItem.ios.tsx [hunk 1]
```diff
@@ -101,6 +106,10 @@ const ImageItem = ({
     scrollValueY.setValue(offsetY);
   };
 
+  const onLongPressHandler = (event: GestureResponderEvent) => {
```
Comment (antonKalinin) [inline src/components/ImageItem/ImageItem.ios.tsx:109]: Better wrap it with `useCallback` here

File: src/components/ImageItem/ImageItem.android.tsx [hunk 2]
```diff
@@ -101,6 +107,10 @@ const ImageItem = ({
     scrollValueY.setValue(offsetY);
   };
 
+  const onLongPressHandler = (event: GestureResponderEvent) => {
```
Comment (antonKalinin) [inline src/components/ImageItem/ImageItem.android.tsx:110]: Better wrap it with useCallback here

**Comment by nojimage** [comment]
@antonKalinin thank you for reviewing. I changed it.

**Comment by maksimko** [comment]
👍 All good, thank you.

**Pull request merged**